### PR TITLE
[cli] allow to open external URLs in a new tab

### PR DESCRIPTION
**Pull request by JanKoehnlein**
A follow up on https://github.com/gitpod-io/gitpod/pull/3565.
Allows `gp preview` to open pages in new tabs

### CODE REVIEW

File: components/supervisor/pkg/supervisor/git.go [hunk 1]
```diff
@@ -50,14 +53,22 @@ func (p *GitTokenProvider) GetToken(ctx context.Context, req *api.GetTokenReques
 	}
 	missing := getMissingScopes(req.Scope, scopes)
 	if len(missing) > 0 {
-		message := fmt.Sprintf("An operation requires additional permissions: %s. Please grant these on the [access control page](%s).", strings.Join(missing, ", "), p.workspaceConfig.GitpodHost+"/access-control")
-		_, err = p.notificationService.Notify(ctx, &api.NotifyRequest{
+		message := fmt.Sprintf("An operation requires additional permissions: %s.", strings.Join(missing, ", "))
+		result, err := p.notificationService.Notify(ctx, &api.NotifyRequest{
 			Level:   api.NotifyRequest_INFO,
 			Message: message,
+			Actions: []string{"Open Access Control"},
```
Comment (akosyakov) [inline components/supervisor/pkg/supervisor/git.go:60]: maybe better `Grant Permissions`? i.e. call to action, or at least put in message for instance `Please grant permissions and try again`.

Comment (JanKoehnlein) [reply components/supervisor/pkg/supervisor/git.go:58]: I extended the message. I think it's good if the user understands that clicking the button does not yet grant the permissions.

**Comment by JanKoehnlein** [comment]
When testing: not that the info dialog currently comes very late sometimes minutes (!) after the notification. I think we're waiting for OpenVSX before we actually start the Gitpod extension and thus register for notifications. Could be related to https://github.com/gitpod-io/gitpod/pull/3643

**Review by akosyakov — APPROVED** [review]
worked well for me

**Pull request merged**